relayer/src/config.rs

```diff
@@ -82,6 +104,25 @@ impl Config {
     pub fn find_chain_mut(&mut self, id: &ChainId) -> Option<&mut ChainConfig> {
         self.chains.iter_mut().find(|c| c.id == *id)
     }
+
+    /// Returns true if the given pair [`PortId`] [`ChannelId`] on
+    /// [`ChainId`] is allowed.
+    /// Returns false otherwise.
+    pub fn channel_allowed(
+        &self,
+        chain_id: &ChainId,
+        port_id: &PortId,
+        channel_id: &ChannelId,
+    ) -> bool {
+        match self.find_chain(chain_id) {
+            None => false,
+            Some(chain_config) => match &chain_config.channel_filter {
+                ChannelFilter::Allow(spec) => spec.contains(&(port_id.clone(), channel_id.clone())),
+                ChannelFilter::Deny(spec) => !spec.contains(&(port_id.clone(), channel_id.clone())),
+                ChannelFilter::AllowAll => true,
+            },
```

Comment:
Can we move this into a `is_allowed` method on `ChannelFilter`?

Reply:
9ae7235